config/initializers/assets.rb

```diff
@@ -11,3 +11,7 @@
 # application.js, application.scss, and all non-JS/CSS in app/assets folder
 # are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+%w( home issues users application ).each do |controller|
+  Rails.application.config.assets.precompile += ["#{controller}.js.coffee"]
```

Comment:
//= require home.js in application.js and other files.
it will precompile everything automatically